packages/client/hmi-client/src/workflow/ops/model-config/tera-model-config-table.vue

```diff
@@ -0,0 +1,206 @@
+<template>
+	<Datatable
+		:value="data"
+		v-model:expanded-rows="expandedRows"
+		dataKey="id"
+		:row-class="rowClass"
+		class="p-datatable-sm"
+		:class="{ 'hide-header': hideHeader }"
+	>
+		<Column expander style="width: 5rem" />
+		<Column field="id" header="ID" style="width: 10%"></Column>
+		<Column field="name" header="Name" style="width: 15%"></Column>
+		<Column field="type" header="Value type" style="width: 10%">
+			<template #body="slotProps">
+				<Button
+					text
+					v-if="slotProps.data.type === ParamType.MATRIX"
+					icon="pi pi-table"
+					@click="openMatrixModal(tableType, slotProps.data.id)"
+				/>
+				<span v-else-if="slotProps.data.type === ParamType.EXPRESSION">Expression</span>
+				<Dropdown
+					v-else
+					:model-value="slotProps.data.type"
+					:options="typeOptions"
+					optionLabel="label"
+					optionValue="value"
+					placeholder="Select a parameter type"
+					@update:model-value="(val) => changeType(slotProps.data.value, val)"
+				/>
```

Comment:
When I switch from constant => distribution, it seemed to wipe out my existing distribution min/max values.

Reply:
The way we distinguish between a constant vs a distribution as far as I know is if the parameter has a distribution key on it.  When I switch between constant vs distribution it adds/deletes the key for distribution which is why it resets.

Reply:
Okay - let's flag this as a FIXME as we may need to revisit this behaviour later.